lib/data/locationConstraintParser.js

```diff
@@ -9,10 +9,10 @@ export default function parseLC() {
 
     Object.keys(config.locationConstraints).forEach(location => {
         const locationObj = config.locationConstraints[location];
-        if (locationObj.type === 'mem') {
+        if (locationObj.type === 'mem' || locationObj.type === 'dataMem') {
```

Comment:
I thought we were going to leave this alone and just change the implName in the data wrapper to "mem" and "file"?